index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./styles.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Sharp:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Rounded:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"/>
+    <!-- <link rel="stylesheet" href="https://kit.fontawesome.com/058c29e44e.css" crossorigin="anonymous"> -->
```

Comment:
I think you should delete the unused code to keep your the code clean

Reply:
Great catch @newhoteng and awesome use of inline comments 👍 You have done very well. However, I think your comment could be tweaked just a little bit to make it sound more like a suggetion 😉 . Check this out: "I think it is a good idea to remove commented pieces of code from your code base. This keeps your the code clean and easier to read" You can add an emoji for a nice touch of friendliness to ensure that the receiver of your review maintains a positive energy and does not feel discouraged 💪